js/generateCard.js

```diff
@@ -0,0 +1,35 @@
+import {similarAdvertisements, syncTypesToRussian} from './data.js';
+import {checkValueOfPhoto, checkValueOfElement, inclineTermonOfRooms, inclineTermonOfGuests} from './util.js';
+
+const addCardTemplate = document.querySelector('#card').content;
+const CardTemplate = addCardTemplate.querySelector('.popup');
+
+similarAdvertisements.forEach((el) => {
```

Comment:
можешь вот так вот сделать
деструктуризация
```suggestion
similarAdvertisements.forEach(({offer, author}) => {
```